src/Core/src/Platform/Android/ImageSourcePartExtensions.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.Maui.Platform
 {
-	internal static class ImageSourcePartExtensions
+	public static class ImageSourcePartExtensions
```

Comment:
This change aligns all platforms (in Tizen this extension method was already public in .NET 7) and allows a way to load images without dependencies on the VirtualView.